coderd/members_test.go

```diff
@@ -54,6 +57,12 @@ func TestListMembers(t *testing.T) {
 
 	t.Run("OK", func(t *testing.T) {
 		t.Parallel()
+
+		// TODO: we should not be returning the prebuilds user in OrganizationMembers, and this is not returned in dbmem.
```

Comment:
I see we exclude system users in users.sql. I think now is the right time to exclude them from these queries as well.